docs/iris/src/userguide/saving_iris_cubes.rst

```diff
@@ -106,3 +107,32 @@ Such a custom saver will need be written to meet the needs of the file format an
 
 Implementing a bespoke saver is out of scope for the user guide.
 
+Caching cubes and cube lists to pickle files
+--------------------------------------------
+
+It should always be possible to create a temporary cache file containing a cube or cube list using the Python `Pickle <https://docs.python.org/2/library/pickle.html>`_ functionality. This can be useful when the cube or cube list has been lazily loaded so the pickle file itself will contain only a reference to the data in the original files. In this state writing, and subsequently reading, a pickle file is very fast.
+
+.. warning::
+
+    Pickle files are not a portable file format, there is no guarantee that pickle files will work if there are any differences between the python/iris environment/version.
+
+    If a pickle file contains lazily loaded cubes then any changes to the files that contain the data will break the ability of the cubes to read its data.
+
+The pickle file is especially useful for code development, where the same data files might need to be loaded repeatedly. Another common use case is parent-child processes where a single process loads data and then a number of subsequent processes can quickly access the data in order to run parallel processing tasks.
+
+A quick example of saving and reading a pickle file is:
+
+.. code-block:: python
+    # import pickle: in python 2.7 cPickle is faster
```

Comment:
What about py35?

In general it is best not to have entire commented lines of code within a code example. In this case it would be better to discuss `pickle` vs `cPickle` in a text section before this code example.